crates/re_data_ui/src/instance_path.rs

```diff
@@ -23,11 +23,13 @@ impl DataUi for InstancePath {
 
         let Some(components) = store.all_components(&query.timeline, entity_path) else {
             if ctx.store_db.entity_db.is_known_entity(entity_path) {
-                ui.label(format!(
-                    "No components in entity {:?} on timeline {:?}",
-                    entity_path,
-                    query.timeline.name()
-                ));
+                ui.label(
+                    egui::RichText::new(format!(
+                        "No components logged on timeline {:?}",
+                        query.timeline.name()
+                    ))
+                    .color(egui::Color32::KHAKI),
```

Comment:
Maybe use `ReUi::warning_text` or add a new `ReUi::surprising_text`. Point it: less hard-coded colors, more use of `re_ui` :)

Reply:
Good call!